evaluation.md

```diff
@@ -5,6 +5,25 @@
 _Describe each change from your ideal syntax to the syntax you implemented, and
 describe_ why _you made the change._
 
+We could not implement our State syntax without more parentheses or brackets, so we changed it to `state ("StateName")` followed by the rules in braces.
```

Comment:
Here's a nice stackoverflow post going into some detail on only space delimited functions. http://stackoverflow.com/questions/1181533/what-are-the-precise-rules-for-when-you-can-omit-parenthesis-dots-braces-f